master/custom/factories.py

```diff
@@ -557,6 +557,15 @@ class Windows64Build(BaseWindowsBuild):
     factory_tags = ["win64"]
 
 
+class Windows64BigmemBuild(BaseWindowsBuild):
+    buildersuffix = ".bigmem"
+    buildFlags = ["-p", "x64"]
+    testFlags = ["-p", "x64", "-j8", "-M24g", "-u-all"]
```

Comment:
I added `-j8` here, does that duplicate `parallel_tests` or should one of them be a different value?

Reply:
Yes, it's a duplicate.  It should probably be set by the worker rather than the builder (as in, leave out `-j` here), in case another worker does this build and wants a different parallelization value.